storage/store.go

```diff
@@ -367,9 +367,20 @@ func (s *Store) String() string {
 
 // Start the engine, set the GC and read the StoreIdent.
 func (s *Store) Start() error {
-	// Start engine (i.e. open and initialize RocksDB database).
-	if err := s.engine.Start(); err != nil {
-		return err
+	if s.Ident.NodeID == 0 {
```

Comment:
Might want to add a comment as to what this would imply: "If NodeID != 0, then the store has already been started."